tests/bin/get_att.c

```diff
@@ -9,7 +13,15 @@ int main(void) {
 
     ssize_t size = get_att(nonce, sizeof(nonce), buf, sizeof(buf), &technology);
 
-    // TODO: Currently this should return ENOSYS. However, this will change
-    // as get_att() is implemented for each technology.
-    return !(size == -1 && errno == ENOSYS);
+    if (technology == NO_KEEP)
+        return !(size == -1 && errno == ENOSYS);
+
+    else if (technology == SEV_TECH)
+        return !(size >= 0);
+
+    else if (technology ==  SGX_TECH)
+        return !(size >= 0);
+
+    else
+        return 1;
```

Comment:
I think we want to invert these branches. Match on the `size` value first. Then `errno` or `technology`. You `MUST NOT` inspect `technology` if an error occurred.

Reply:
That is very reasonable.